src/loaders/svg/tvgSvgSceneBuilder.cpp

```diff
@@ -80,7 +80,7 @@ static unique_ptr<LinearGradient> _applyLinearGradientProperty(SvgStyleGradient*
             stops[i].r = colorStop->r;
             stops[i].g = colorStop->g;
             stops[i].b = colorStop->b;
-            stops[i].a = (colorStop->a * opacity) / 255.0f;
+            stops[i].a = static_cast<uint8_t>((colorStop->a * opacity) / 255.0f);
```

Comment:
We dont need a float operation (int * int / int)

Reply:
I've changed 255.0f -> 255, but I've kept casting, since opacity is int